net/p910nd/files/p910nd.hotplug

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+user=
+
+. /lib/functions.sh
```

Comment:
As an alternative to this, I've seen  ```#!/bin/sh /etc/rc.common``` since rc.common references functions.sh.

Reply:
That doesn't work for hotplug scripts IIRC -- I think they are sourced not executed (at least that's how it used to be).

Reply:
Ah OK